app/harvesters/open_alex/open_alex_references_converter.py

```diff
@@ -0,0 +1,129 @@
+from typing import AsyncGenerator
+from app.db.daos.contributor_dao import ContributorDAO
+from app.db.models.concept import Concept
+
+from app.db.models.contribution import Contribution
+from app.db.models.contributor import Contributor
+from app.db.models.title import Title
+from app.db.models.reference import Reference
+from app.db.models.abstract import Abstract
+from app.db.session import async_session
+
+from app.harvesters.abstract_references_converter import AbstractReferencesConverter
+from app.harvesters.json_harvester_raw_result import JsonHarvesterRawResult
+from app.harvesters.open_alex.open_alex_document_type_converter import (
+    OpenAlexDocumentTypeConverter,
+)
+
+
+class OpenAlexReferencesConverter(AbstractReferencesConverter):
+    """
+    Converts raw data from OpenAlex to a normalised Reference object
+    """
+
+    async def convert(self, raw_data: JsonHarvesterRawResult) -> Reference:
+        """
+        Convert raw data from OpenAlex to a normalised Reference object
+        :param raw_data: raw data from OpenAlex
+        :return: Reference object
+        """
+        json_payload = raw_data.payload
+        new_ref = Reference()
+
+        language = json_payload.get("language", None)
+        new_ref.titles.append(self._title(json_payload, language))
```

Comment:
With ORCID 0000-0002-8156-2229 : 

```
Task exception was never retrieved
future: <Task finished name='openalex_harvester_retrieval_2' coro=<AbstractHarvester.run() done, defined at svp-harvester/app/harvesters/abstract_harvester.py:90> exception=IntegrityError('(sqlalchemy.dialects.postgresql.asyncpg.IntegrityError) <class \'asyncpg.exceptions.NotNullViolationError\'>: null value in column "value" of relation "reference_literal_fields" violates not-null constraint\nDETAIL:  Failing row contains (title, 188, null, en, 91).')>
...
    raise translated_error from error
sqlalchemy.dialects.postgresql.asyncpg.AsyncAdapt_asyncpg_dbapi.IntegrityError: <class 'asyncpg.exceptions.NotNullViolationError'>: null value in column "value" of relation "reference_literal_fields" violates not-null constraint
DETAIL:  Failing row contains (title, 188, null, en, 91).
```

Reply:
It was due a document with title None. In that case, it save in DB an empty string.